packages/resource-scheduler/src/components/ResourceScheduler/ResourceScheduler.js

```diff
@@ -22,7 +34,7 @@ class ResourceScheduler extends React.Component {
     const props = this.props
     return (
       <React.Fragment>
```

Comment:
In this case you can remove React.Fragment. Fragment is only a wrapper that does not create a dom element. See react docs for more information